### DISCUSSION TITLE
Information regarding setup

### DISCUSSION THREAD

**Discussion by Adarsh5harma** [AndrejOrsula/drl_grasping#101] [General]
Hello Andrej Orsula,

I really like the project, does this requires camera to perform the tasks, If  I want to train the robot using this library to reach some goal which is not changing and is fixed does it requires camera for this task. 

Kind Regards,
Adarsh

**Comment by AndrejOrsula** [comment]
Hello @Adarsh3559,

Camera is required to perform the task if visual observation are used (e.g. RGB image, depth map and octree). Instead of visual observations, you can also use environment states (e.g. position of the goal + joint positions/end-effector position), which are easily available inside a simulation but can be more complicated to determine in real world.

If the goal is fixed for the reach task, then visual observations are most likely not necessary. In this particular case, the position of the goal might not even be needed in the observation space.

Note, that the camera can be simulated inside simulation.

Good luck!
